src/instructlab/model/serve.py

```diff
@@ -116,9 +121,6 @@ def serve(
 
     if backend == backends.VLLM:
         # Instantiate the vllm server
-        if vllm_args is None:
```

Comment:
@tiran what was the reason for removing this? `vllm_args` defaults to `None` if the flag is not passed in.

Reply:
Upon playing around with the code unset `--vllm-args` defaults to an empty tuple.